src/simulation/scenario_config.py

```diff
@@ -86,8 +86,8 @@ def get_named_scenarios():
 
     """
     if FAST_FLAG == "debug":
-        n_main_seeds = 1
-        n_other_seeds = 1
+        n_main_seeds = 0
+        n_other_seeds = 0
```

Comment:
Not even 1?
```suggestion
        n_main_seeds = 1
        n_other_seeds = 1
```

Reply:
Forgot to turn that back after developing :innocent: